.run/Frontend-LocalRun.run.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="save-frontend:run --continuous" type="GradleRunConfiguration" factoryName="Gradle">
+  <configuration default="false" name="Frontend-LocalRun" type="GradleRunConfiguration" factoryName="Gradle">
```

Comment:
maybe create a runner for mysql and select JRE-11 there?

Reply:
I guess it was a change from a merge commit 🤔